CoreGraphFramework/test/unit/src/au/gov/asd/tac/constellation/graph/attribute/BooleanAttributeDescriptionNGTest.java

```diff
@@ -0,0 +1,461 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.graph.attribute;
+
+import au.gov.asd.tac.constellation.graph.GraphReadMethods;
+import au.gov.asd.tac.constellation.graph.NativeAttributeType;
+import au.gov.asd.tac.constellation.graph.StoreGraph;
+import au.gov.asd.tac.constellation.graph.locking.ParameterReadAccess;
+import au.gov.asd.tac.constellation.graph.locking.ParameterWriteAccess;
+import au.gov.asd.tac.constellation.graph.utilities.MultiValueStore;
+import static org.testng.Assert.*;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author arcturus
+ */
+public class BooleanAttributeDescriptionNGTest {
+
+    BooleanAttributeDescription instance;
+
+    public BooleanAttributeDescriptionNGTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+        instance = new BooleanAttributeDescription();
+        instance.setCapacity(1);
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+    }
+
+    /**
+     * Test of getName method, of class BooleanAttributeDescription.
+     */
+    @Test
+    public void testGetName() {
+        String expResult = "boolean";
+        String result = instance.getName();
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of getNativeClass method, of class BooleanAttributeDescription.
+     */
+    @Test
+    public void testGetNativeClass() {
+        Class expResult = boolean.class;
+        Class result = instance.getNativeClass();
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of getNativeType method, of class BooleanAttributeDescription.
+     */
+    @Test
+    public void testGetNativeType() {
+        NativeAttributeType expResult = NativeAttributeType.BOOLEAN;
+        NativeAttributeType result = instance.getNativeType();
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of getDefault method, of class BooleanAttributeDescription.
+     */
+    @Test
+    public void testGetDefault() {
+        Object expResult = false;
+        Object result = instance.getDefault();
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of setDefault method, of class BooleanAttributeDescription.
+     */
+    @Test
+    public void testSetDefault() {
+        Object value = null;
+        instance.setDefault(value);
```

Comment:
Should this have an assert to check if it was set correctly?

Reply:
Updated thanks so much for the feedback